kafka.go

```diff
@@ -94,7 +94,7 @@ func (om *OpenMock) startKafka() {
 	if err := om.configKafka(); err != nil {
 		logrus.WithFields(logrus.Fields{
 			"err": err,
-		}).Errorf("failed to config kafka")
+		}).Fatal("failed to config kafka")
```

Comment:
Here we are trying to `Fatal` the openmock process, so that docker-compose or kubernetes can retry on the crashed container.

Because if we set kafka enabled to true, but kafka brokers are not ready, it doesn't make sense to continue the process.